test/request.go

```diff
@@ -50,6 +50,22 @@ func Retrying(rc spoof.ResponseChecker, codes ...int) spoof.ResponseChecker {
 	}
 }
 
+// IsStatusCode checks that the response code is equal to the given one.
+func IsStatusCode(code int) spoof.ResponseChecker {
```

Comment:
Would it be useful, to extend this to `codes []int` and check if any of matches? E.g. below we can check for 200 and 201.

Reply:
In this case, yes.

I won't change the Matcher below though. I'd like us to write as tight checks as possible (I'm in the process of cleaning them all up). If we expect a 201, we should explicitly wait for that in the test.